src/opnsense/service/conf/actions.d/actions_filter.conf

```
@@ -98,7 +98,7 @@ message:request pf rules
 [diag.top]
 command:/usr/local/opnsense/scripts/filter/pftop.py
 type:script_output
-cache_ttl:30
+cache_ttl:600
```

maybe 5 minutes? the time of a coffee break

```suggestion
cache_ttl:300
```